{
  "id": "CVE-2019-18874",
  "elsa_id": "ELSA-2021-4324",
  "definition_id": "oval:com.oracle.elsa:def:20214324",
  "title": "\nELSA-2021-4324:  python-psutil security update (MODERATE)\n",
  "description": "\n[5.4.3-11]\n- Security fix for CVE-2019-18874: double free because of refcount mishandling\nResolves: rhbz#1772014\n\n",
  "severity": "MODERATE",
  "affected": {
    "family": "unix",
    "platform": "Oracle Linux 8"
  },
  "packages": [
    {
      "name": "python3-psutil",
      "fixed_version": "0:5.4.3-11.el8",
      "arch": "aarch64"
    },
    {
      "name": "python3-psutil",
      "fixed_version": "0:5.4.3-11.el8",
      "arch": "x86_64"
    }
  ],
  "references": [
    {
      "id": "ELSA-2021-4324",
      "source": "elsa",
      "url": "https://linux.oracle.com/errata/ELSA-2021-4324.html"
    },
    {
      "id": "CVE-2019-18874",
      "source": "CVE",
      "url": "https://linux.oracle.com/cve/CVE-2019-18874.html"
    }
  ],
  "issued": "2021-11-16T00:00:00Z",
  "rights": "Copyright 2021 Oracle, Inc."
}
